.prettierignore

```diff
@@ -0,0 +1,2 @@
+modules/locale/tests/locale_test.es6.js
```

Comment:
```suggestion
```

Reply:
Oops, we don't need that line.

Reply:
Do you want to commit this change so we can re-approve?